diff -ur a/usr/bin/gamescope-session b/usr/bin/gamescope-session
--- a/usr/bin/gamescope-session	2023-06-22 20:05:10.901946505 -0700
+++ b/usr/bin/gamescope-session	2023-06-22 20:06:49.143821090 -0700
@@ -5,6 +5,9 @@
 # Window managers sets this to wayland but apps using Gamescope must use x11
 export XDG_SESSION_TYPE=x11
 
+# Bazzite: Enable MangoHUD by default
+export MANGOHUD=1
+
 # Update the enviroment with DESKTOP_SESSION and all XDG variables
 dbus-update-activation-environment --systemd DESKTOP_SESSION `env | grep ^XDG_ | cut -d = -f 1`

